e2e/scripts/st_plotly_chart.py

```diff
@@ -213,3 +215,16 @@
 
 fig = px.density_heatmap(df, x="total_bill", y="tip")
 st.plotly_chart(fig, theme="streamlit")
+
+data = pd.DataFrame((100, 120, 104, 102, 203, 102), columns=["some_col"])
+
+fig = px.line(data, height=100, width=300)
+fig.update_xaxes(visible=False, fixedrange=True)
+fig.update_yaxes(visible=False, fixedrange=True)
+fig.update_layout(annotations=[], overwrite=True)
+fig.update_layout(showlegend=False, margin=dict(t=10, l=10, b=10, r=10))
+st.plotly_chart(
+    fig, config=dict(displayModeBar=False), use_container_width=False, theme=None
```

Comment:
Checks for `use_container_width=False` when height and width are changed.

Reply:
To reword:
This checks that we use do not use the container width when the height and width are changed and no streamlit theme is applied.